abc.templates/base-workflows/contents/guardian-admin.yml

```diff
@@ -129,11 +127,9 @@ jobs:
         env:
           # used by terraform when using the github provider
           TERRAFORM_COMMAND: '${{ inputs.command }}'
-          TERRAFORM_ARGS: '${{ inputs.args }}'
           DIRECTORY: '${{ matrix.working_directory }}'
         shell: 'bash'
         run: |-
           guardian run \
-          -terraform-command="${TERRAFORM_COMMAND}" \
-          -terraform-args="${TERRAFORM_ARGS}" \
-          "${DIRECTORY}"
+          -dir="${DIRECTORY}" \
+          -- ${TERRAFORM_COMMAND}"
```

Comment:
Looks like there's an extra " at the end here

Reply:
fixed.